pom.xml

```diff
@@ -17,15 +17,15 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <signature.api.version>2.1</signature.api.version>
-        <slf4j.version>1.7.21</slf4j.version>
+        <slf4j.version>1.7.25</slf4j.version>
     </properties>
 
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>org.springframework</groupId>
                 <artifactId>spring-framework-bom</artifactId>
-                <version>4.3.12.RELEASE</version>
+                <version>4.3.14.RELEASE</version>
```

Comment:
Noen grunn til å ikke bare pejs på med v5 først som sist?

Reply:
Var bare litt redd for at det førte med seg masse endringer jeg ikke var interessert i å bruke tid på akkurat nå. Meen det er jo ikke verre enn å gjøre et forsøk da 🙃

Reply:
Spring er som Java, ny major-versjon er null endringer.